driver/sqlite3/sqlite3.go

```diff
@@ -91,20 +91,15 @@ func (driver *Driver) Migrate(f file.File, pipe chan interface{}) {
 		return
 	}
 
-	if _, err := tx.Exec(string(f.Content)); err != nil {
-		sqliteErr, isErr := err.(sqlite3.Error)
-
-		if isErr {
-			// The sqlite3 library only provides error codes, not position information. Output what we do know
-			pipe <- errors.New(fmt.Sprintf("SQLite Error (%s); Extended (%s)\nError: %s", sqliteErr.Code.Error(), sqliteErr.ExtendedCode.Error(), sqliteErr.Error()))
```

Comment:
You should use [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) in your favorite editor, or [vim-go ](https://github.com/fatih/vim-go) directly. The build is failing because you removed `sqliteErr, isErr := err.(sqlite3.Error)`, so the sqlite import is now useless. If this code is really to be removed, so import the package anonymously (`_ "github.com/mattn/go-sqlite3"`).
Thanks